docs/visual-basic/index.md

```diff
@@ -47,5 +47,5 @@ Visual Basic is engineered for productively building type-safe and object-orient
  [Get Started Developing with Visual Studio](/visualstudio/ide/get-started-developing-with-visual-studio)  
  Provides links to topics that help you learn about the basics of Visual Studio.  
   
- [.NET Framework Class Library](http://go.microsoft.com/fwlink/?LinkID=227195)  
+ [.NET Framework Class Library](/dotnet/api/?view=netframework-4.7.2)  
```

Comment:
It's best here not to include the version filter (?view=netframework-4.7.2), or this will have to be updated for each .NET Framework release, a detail that's certain to be overlooked.

Reply:
As far as I remember, in other similar cases, the link text was changed to ".NET API Browser":
```
[.NET API Browser](/dotnet/api)
```

Reply:
Yes, sorry about omitting that detail, @florinciubotariu, and thanks for pointing it out, @pkulikov.  I'll revise my previous comments.

Reply:
Also, given that the page is in our repo, you could probably use a relative path to https://github.com/dotnet/docs/blob/master/api/index.md